bundle.yaml.j2

```diff
@@ -26,7 +26,7 @@ applications:
     series: jammy
   identity-platform-login-ui-operator:
     charm: identity-platform-login-ui-operator
-    revision: 39
+    revision: 74
```

Comment:
we need to make sure that this is the revision with the image before the latest release, aka no OpenFGA

that is, unless u want to make sure all works with the latest one

@nsklikas might have a better idea about it

Reply:
The openfga changes are not released yet